packages/packages/v1-adapters/src/dispatchers/dispatchers.ts

```diff
@@ -36,3 +36,13 @@ export function openRoute( route: string ): Promise<void> {
 		return Promise.reject( e );
 	}
 }
+
+export function isRouteActive( route: string ): boolean {
+	const extendedWindow = window as unknown as ExtendedWindow;
+
+	if ( ! extendedWindow.$e?.routes?.isPartOf ) {
```

Comment:
Why do you need it? the returned check covers it.

Reply:
Done